.github/workflows/test-with-edm.yml

```diff
@@ -8,33 +8,31 @@ on:
     - cron:  '0 0 * * 5'
 
 env:
-  INSTALL_EDM_VERSION: 3.3.1
+  INSTALL_EDM_VERSION: 3.5.0
 
 jobs:
 
   # Test against EDM packages
   test-with-edm:
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-latest, windows-latest]
+        os: [ubuntu-latest, windows-latest]
         runtime: ['3.6', '3.8']
     runs-on: ${{ matrix.os }}
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Cache EDM packages
-        uses: actions/cache@v2
+        uses: actions/cache@v3
         with:
           path: ~/.cache
           key: ${{ runner.os }}-${{ hashFiles('etstool.py') }}
       - name: Set up EDM
-        uses: enthought/setup-edm-action@v1
+        uses: enthought/setup-edm-action@v2
         with:
           edm-version: ${{ env.INSTALL_EDM_VERSION }}
       - name: Install click to the default EDM environment
         run: edm install -y wheel click coverage
       - name: Install test environment
         run: edm run -- python etstool.py install --runtime=${{ matrix.runtime }}
       - name: Run tests
-        uses: GabrielBB/xvfb-action@v1
```

Comment:
There shouldn't be any need for a framebuffer in the tests.

Reply:
There is UI code in the library, but it is untested and we don't have any UI code installed (either Wx or Qt) so this is fine.